llvm/lib/Target/AArch64/AArch64RegisterInfo.td

```diff
@@ -996,6 +996,16 @@ let Namespace = "AArch64" in {
   def psub1 : SubRegIndex<16, -1>;
 }
 
+class PPRorPNRClass : RegisterClass<
+                                  "AArch64",
+                                  [ nxv16i1, nxv8i1, nxv4i1, nxv2i1, nxv1i1 ], 16,
+                                  (add PPR, PNR)> {
+  let Size = 16;
+}
+def PPRorPNR    : PPRorPNRClass;
+def PPRorPNRAsmOpAny   : PPRAsmOperand<"PPRorPNRAny",   "PPRorPNR",     0>;
```

Comment:
You need to add a case statement for `Match_InvalidSVEPPRorPNRAnyReg to:
* `AArch64AsmParser::showMatchError`
* `AArch64AsmParser::MatchAndEmitInstruction`